baseapp/baseapp.go

```diff
@@ -172,7 +170,6 @@ func (app *BaseApp) NewContext(isCheckTx bool, header abci.Header) sdk.Context {
 
 // Implements ABCI
 func (app *BaseApp) Info(req abci.RequestInfo) abci.ResponseInfo {
-
```

Comment:
I actually prefer these leading spaces when there is a long function header.
I think it is personal preference, and maybe you can respect the code style in the repo. These lines came from Jae, and I agree with this style.